WordPressUI/Extensions/UIImageView+Gravatar.swift

```diff
@@ -51,12 +51,49 @@ extension UIImageView {
     ///     - placeholderImage: Image to be used as Placeholder
     ///
     @objc
-    public func downloadGravatarWithEmail(_ email: String, rating: GravatarRatings = .`default`, placeholderImage: UIImage = .gravatarPlaceholderImage) {
-        let gravatarURL = gravatarUrl(for: email, size: gravatarDefaultSize(), rating: rating.stringValue())
+    public func downloadGravatarWithEmail(_ email: String, rating: GravatarRatings = .default, placeholderImage: UIImage = .gravatarPlaceholderImage) {
+        let gravatarURL = gravatarUrl(for: email, size: gravatarDefaultSize(), rating: rating)
 
+        listenForGravatarChanges()
         downloadImage(from: gravatarURL, placeholderImage: placeholderImage)
     }
 
+    /// Configures the UIImageView to listen for changes to the gravatar it is displaying
+    private func listenForGravatarChanges() {
+        guard gravatarObserver == nil else {
+            return
+        }
+
+        gravatarObserver = NotificationCenter.default.addObserver(forName: .GravatarImageUpdateNotification, object: nil, queue: nil) { [weak self] (notification) in
+            guard let userInfo = notification.userInfo,
+                let email = userInfo["email"] as? String,
+                let image = userInfo["image"] as? UIImage,
```

Comment:
Suggestion: It'd be neat to turn both `email` / `image` into a constant, so that we don't need to have the whole string in two spots (repetition / fragile-ness).